pkg/cloud/openstack.go

```diff
@@ -0,0 +1,138 @@
+package cloud
+
+import (
+	"bytes"
+	_ "embed"
+
+	appsv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	k8sYaml "k8s.io/apimachinery/pkg/util/yaml"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+//go:embed assets/openstack/config.yaml
+var openstackConfig []byte
+
+//go:embed assets/openstack/deployment.yaml
+var openstackDeployment []byte
+
+//go:embed assets/openstack/rbac/serviceaccount.yaml
+var openstackServiceAccount []byte
+
+//go:embed assets/openstack/rbac/role.yaml
+var openstackRole []byte
+
+//go:embed assets/openstack/rbac/rolebinding.yaml
+var openstackRoleBinding []byte
+
+//go:embed assets/openstack/rbac/clusterrolebinding.yaml
+var openstackClusterRoleBinding []byte
+
+func GetOpenStackResources() []client.Object {
+	config := newConfigResource()
+	if config == nil {
+		return nil
+	}
+
+	deployment := newDeploymentResource()
+	if config == nil {
```

Comment:
s/config/deployment/

Cut/paste errors all the way down :)

Reply:
yep :(